providers/capa/standard/capa_suite_test.go

```diff
@@ -8,56 +8,73 @@ import (
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-
 	cr "sigs.k8s.io/controller-runtime/pkg/client"
 
 	"github.com/giantswarm/clustertest"
 	"github.com/giantswarm/clustertest/pkg/application"
 	"github.com/giantswarm/clustertest/pkg/logger"
 	"github.com/giantswarm/clustertest/pkg/utils"
 	"github.com/giantswarm/clustertest/pkg/wait"
+
+	"github.com/giantswarm/cluster-test-suites/common"
 )
 
 const KubeContext = "capa"
 
 var (
+	ctx       context.Context
 	framework *clustertest.Framework
 	cluster   *application.Cluster
 )
 
 func TestCAPAStandard(t *testing.T) {
-	var err error
-	ctx := context.Background()
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "CAPA Standard Suite")
+}
 
+var _ = BeforeSuite(func() {
+	ctx = context.Background()
 	logger.LogWriter = GinkgoWriter
 
+	var err error
 	framework, err = clustertest.New(KubeContext)
-	if err != nil {
-		panic(err)
+	Expect(err).NotTo(HaveOccurred())
+
+	cluster = setUpWorkloadCluster()
+
+	common.Framework = framework
+	common.Cluster = cluster
+})
+
+func setUpWorkloadCluster() *application.Cluster {
+	cluster, err := framework.LoadCluster()
+	Expect(err).NotTo(HaveOccurred())
+	if cluster != nil {
+		return cluster
```

Comment:
Might be worth logging out here that we've loaded an existing cluster and the name of that cluster.

Reply:
Added a log in there. Thanks